PlayerScripts/PlayerScriptsPlace/StarterPlayer/StarterPlayerScripts/CameraScript/RootCamera.rbxmx

```diff
@@ -294,6 +294,27 @@ local function CreateCamera()
 
 	function this:Update()
 	end
+	
+	function this:ApplyVRPosition()
+		--we only want this to happen in first person VR
+		local player = game.Players.LocalPlayer
+		if not player or not player.Character 
+				or not player.Character:FindFirstChild("HumanoidRootPart") 
+				or not player.Character.HumanoidRootPart:FindFirstChild("RootJoint") then
+			return
+		end
+		
+		if not UserInputService.VREnabled or not this:IsInFirstPerson() then
```

Comment:
It seems weird we are doing cframe adjustments in a function called ApplyVRPosition when VR is not enabled